sdk/core/core-amqp/src/util/constants.ts

```diff
@@ -32,7 +32,7 @@ export const eventHub = "eventhub";
 export const cbsEndpoint = "$cbs";
 export const cbsReplyTo = "cbs";
 export const operationPutToken = "put-token";
-export const aadEventHubsAudience = "https://eventhubs.azure.net/";
+export const aadEventHubsScope = "https://eventhubs.azure.net//.default";
```

Comment:
Instead of removing the "audience"  variation, lets only add the "scope" variation. The old "audience" is still useful if anyone wants to use this library directly with say ms-rest-nodeauth.

So similar changes to service bus in the next line